Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -41,6 +41,15 @@ open class Player: AVPlayerViewController {
         }
     }
 
+    override open var preferredFocusEnvironments: [UIFocusEnvironment] {
+        let subviews = contentOverlayView?.subviews.first?.subviews
+        if let button = subviews?.first(where: { $0 is UIButton }), button.alpha > 0.1 {
```

Comment:
I got the gist we're trying to return the SkipButton as the preferred focus environment, but I didn't quite get why there's this `button.alpha > 0.1` check. Could you guys shine some light on this matter, please?

Reply:
When we hide or show a action button we do it with the alpha value animated, so if the alpha is greater then 0.1 it is being showed. If we fix it in 1.0 it could have any bugs related to animation.

Reply:
Then I guess we can extract it to some `isVisible:button:` function (or something in that line), just so the code's intent is clear.

Reply:
I guess what is missing here is to actually hide the button view with `isHidden = true`. A hidden view should not be processed by the Focus Engine, therefore making this check obsolete.

Reply:
We tried an approach with hidden and it has some inconsistencies, we need to work with alpha here because of the animations of show/hide, if we do not do this the player do not know when to come back the focus to the seek bar when the button is hiding.

Reply:
Done